CONTRIBUTING.md

```diff
@@ -18,18 +18,18 @@ Can you read this? Then you! Simply consult the short [style guidelines](#style-
 If the below instructions are foreign to you, consider giving a reading to the [`git` module](/general/beginner/git/README.md) to "*git*" familiar with git! The git workflow has some colorful jargon but none of the concepts should be new to you if you have used a computer before. If you *haven't* used a computer before, then it might be best to [start somewhere more basic](https://www.pcworld.com/article/2918397/how-to-get-started-with-linux-a-beginners-guide.html).
 
 #### Where do I put my content? 
-* There are two primary locations for contributions, this base repository or the gh-pages branch, which renders files on our [Github Pages website](https://nrel.github.io/HPC/). The gh-pages branch is for contributions that require more explanations/are highly verbose, whereas this repo should be predominantly composed of scripts/source code/executables that _do_ something on the HPC systems. 
+There are two primary locations for contributions, this base repository or the gh-pages branch, which renders files on our [Github Pages website](https://nrel.github.io/HPC/). The gh-pages branch is for contributions that require more explanations/are highly verbose, whereas this repo should be predominantly composed of scripts/source code/executables that _do_ something on the HPC systems. 
```

Comment:
There are two primary locations for contributions: either this base repository, or the gh-pages branch, which renders files on our [Github Pages website](https://nrel.github.io/HPC/). The gh-pages branch is for contributions that require more explanations and/or are highly verbose, whereas this repo should be predominantly composed of scripts/source code/executables that _do_ something on the HPC systems.